node/flags/flags.go

```diff
@@ -156,17 +156,11 @@ var (
 		Usage:    "Password to decrypt ecdsa private key",
 		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "ECDSA_KEY_PASSWORD"),
 	}
-	BlsOperatorStateRetrieverFlag = cli.StringFlag{
-		Name:     common.PrefixFlag(FlagPrefix, "bls-operator-state-retriever"),
-		Usage:    "Address of the BLS Operator State Retriever",
-		Required: true,
-		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "BLS_OPERATOR_STATE_RETRIVER"),
-	}
-	EigenDAServiceManagerFlag = cli.StringFlag{
-		Name:     common.PrefixFlag(FlagPrefix, "eigenda-service-manager"),
-		Usage:    "Address of the EigenDA Service Manager",
-		Required: true,
-		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "EIGENDA_SERVICE_MANAGER"),
```

Comment:
please do not ever do this. this will break users and they will scream at us. flag breaking changes should be announced way ahead of time, and should have been deprecated for a while, and any PRs removing flags (breaking API change!) should have its title with an `!` like `feat!: eth client address directory` to mark the breaking change (to be clear we should not do this right now, but the ! is useful to flag to the release captain that a MAJOR version bump is needed)